packages/wp-now/src/output.ts

```diff
@@ -0,0 +1,5 @@
+function shouldOutput() {
+	return process.env.NODE_ENV !== 'test';
+}
+
+export const output = shouldOutput() ? console : null;
```

Comment:
I love it, thank you! Later on let's export a function to disable that manually for the VS Code extension